messenger_users/views.py

```diff
@@ -79,6 +80,35 @@ def confirm_linkedin_profile(request):
 # def browse_saved_jobs(request):
 #     pass
 
+def search_linkedIn_profiles(request):
```

Comment:
Add docstring, check the other pull request for instructions

Reply:
Also test this function by using chatfuel and your messenger account to verify that this works